MauiSample/Presentation/Views/TabA.xaml

```diff
@@ -11,7 +11,7 @@
 
             <Style x:Key="MaterialTabStyle" TargetType="tabs:MaterialUnderlinedTabItem">
                 <Setter Property="SelectedTabColor" Value="{StaticResource Primary}" />
-                <Setter Property="SelectedTabTextColor" Value="{StaticResource Secondary}" />
+                <!--<Setter Property="SelectedTabTextColor" Value="{StaticResource Secondary}" />-->
```

Comment:
why is this commented ?

Reply:
To fix a build error:
```
Cannot resolve property "SelectedTabTextColor"
on type "MaterialUnderlinedTabItem
(property missing or missing accessors)".
```

Reply:
AFAICS the error message is correct, right? `SelectedTabTextColor` is only available in `BottomTabItem`, but not in `MaterialUnderlinedTabItem` (both inherit from `TabTextItem`, directly or indirectly).
Not sure why this was missed with net6.0. Probably improved diagnostics in net8.0, I guess ...